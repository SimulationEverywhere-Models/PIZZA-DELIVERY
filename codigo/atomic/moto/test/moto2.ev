00:00:01:000 Carga_mi 3
00:00:02:000 Carga_mi 4
00:00:03:000 Carga_mi 5
