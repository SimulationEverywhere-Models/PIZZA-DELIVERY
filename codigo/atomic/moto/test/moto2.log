Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para moto(02)
Mensaje D / 00:00:00:000 / moto(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / carga_mi /      3.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / carga_mi /      3.00000 para moto(02)
Mensaje D / 00:00:01:000 / moto(02) / 00:00:03:000 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje X / 00:00:02:000 / Root(00) / carga_mi /      4.00000 para top(01)
Mensaje X / 00:00:02:000 / top(01) / carga_mi /      4.00000 para moto(02)
Mensaje D / 00:00:02:000 / moto(02) / 00:00:02:000 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje X / 00:00:03:000 / Root(00) / carga_mi /      5.00000 para top(01)
Mensaje X / 00:00:03:000 / top(01) / carga_mi /      5.00000 para moto(02)
Mensaje D / 00:00:03:000 / moto(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:03:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:03:000 / Root(00) para top(01)
Mensaje * / 00:00:03:000 / top(01) para moto(02)
Mensaje Y / 00:00:03:000 / moto(02) / ocupada_mi /      1.00000 para top(01)
Mensaje D / 00:00:03:000 / moto(02) / 00:00:00:240 para top(01)
Mensaje Y / 00:00:03:000 / top(01) / ocupada_mi /      1.00000 para Root(00)
Mensaje D / 00:00:03:000 / top(01) / 00:00:00:240 para Root(00)
Mensaje * / 00:00:03:240 / Root(00) para top(01)
Mensaje * / 00:00:03:240 / top(01) para moto(02)
Mensaje Y / 00:00:03:240 / moto(02) / entrega_mi /      3.00000 para top(01)
Mensaje D / 00:00:03:240 / moto(02) / 00:00:00:060 para top(01)
Mensaje Y / 00:00:03:240 / top(01) / entrega_mi /      3.00000 para Root(00)
Mensaje D / 00:00:03:240 / top(01) / 00:00:00:060 para Root(00)
Mensaje * / 00:00:03:300 / Root(00) para top(01)
Mensaje * / 00:00:03:300 / top(01) para moto(02)
Mensaje Y / 00:00:03:300 / moto(02) / entrega_mi /      4.00000 para top(01)
Mensaje D / 00:00:03:300 / moto(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:03:300 / top(01) / entrega_mi /      4.00000 para Root(00)
Mensaje D / 00:00:03:300 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:03:330 / Root(00) para top(01)
Mensaje * / 00:00:03:330 / top(01) para moto(02)
Mensaje Y / 00:00:03:330 / moto(02) / entrega_mi /      5.00000 para top(01)
Mensaje D / 00:00:03:330 / moto(02) / 00:00:00:330 para top(01)
Mensaje Y / 00:00:03:330 / top(01) / entrega_mi /      5.00000 para Root(00)
Mensaje D / 00:00:03:330 / top(01) / 00:00:00:330 para Root(00)
Mensaje * / 00:00:03:660 / Root(00) para top(01)
Mensaje * / 00:00:03:660 / top(01) para moto(02)
Mensaje Y / 00:00:03:660 / moto(02) / libre_mi /      1.00000 para top(01)
Mensaje D / 00:00:03:660 / moto(02) / ... para top(01)
Mensaje Y / 00:00:03:660 / top(01) / libre_mi /      1.00000 para Root(00)
Mensaje D / 00:00:03:660 / top(01) / ... para Root(00)
