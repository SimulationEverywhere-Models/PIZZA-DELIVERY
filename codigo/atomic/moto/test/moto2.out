00:00:03:000 ocupada_mi 1
00:00:03:240 entrega_mi 3
00:00:03:300 entrega_mi 4
00:00:03:330 entrega_mi 5
00:00:03:660 libre_mi 1
